alembic/versions/347ba3ac054f_update_image_attrs.py

```diff
@@ -22,7 +26,10 @@
 
 
 def upgrade():
-    imgs = StoredFile.query.filter_by(size=None)
+    connection = op.get_bind()
+    Session = sessionmaker(bind=connection.engine)
+    session = Session(bind=connection)
+    imgs = session.query(StoredFile).filter_by(size=None).options(load_only("id", "name", "title"))
```

Comment:
@jace can do that with `op.execute()` but then have to define the schema for stored file in this migration file itself. I wanted to avoid it. so creating a session and using that so that I can use the ORM.